app.js

```diff
@@ -4,13 +4,12 @@ const path = require('path');
 const app = express();
 
 const PORT = process.env.PORT || 8080;
-const JS_DIR = 'js';
-const JS_MOUNT_PATH = '/js';
 const MODELS_DIR = 'models';
 const MODELS_MOUNT_PATH = '/models';
 const INDEX_PATH = path.join(__dirname + '/index.html');
+const PUBLIC_DIR = 'public';
 
-app.use(JS_MOUNT_PATH, express.static(JS_DIR));
+app.use(express.static(PUBLIC_DIR));
 app.use(MODELS_MOUNT_PATH, express.static(MODELS_DIR));
```

Comment:
@misterpeddy @amirhmk I've applied Pedram's nitpick and removed what seemed redundant (the app doesn't need to access the js modules directly as `index.html` already includes `main.js` which imports the rest.
So unless there's an actual need to have the modules accessible on `http://.../js` then we can merge this and start the next iteration?

Reply:
ah amazing, thank you!